site/_posts/2017-03-12-marching-ahead-embedded-ansible.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Last Week in ManageIQ: March-ing ahead with the embedded Ansible integration
+author: lgalis
+date: 2017-03-12 21:33:30 -04:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Laura](https://github.com/lgalis) here bringing you this week's edition of Last Week in ManageIQ!
+
+## Featured
+A busy week again with a lot of progress towards our embedded Ansible Tower features, along with refactoring of existing code, as we move closer to the 
+development complete milestone for our Fine release. Last week, there were 293 commits in 184 PRs across our combined repositories: 
+ - 73 PRs in the [manageiq](manageiq PRs merged) repo
+ - 84 in the [manageiq-ui-classic](manageiq-ui-classic PRs merged) repo
+ - 17 in the [manageiq-ui-service](manageiq-ui-service PRs merged) repo 
+with the rest in the various providers' repos.
+ 
+ 
+## Improved
+
+In this [PR](https://github.com/ManageIQ/manageiq/pull/14239), [Jason](https://github.com/Fryguy) moved dynamic method building into DrbRemoteInvoker,
+allowing it to run in the launched automate ruby process directly and not over DRb, preventing random DRb thread issues.
+
+The event collection for the datawarehouse provider now has field to event streams to keep track of the ext_management_system that carried the event in,  
+as added by [Mooli](https://github.com/moolitayer) [here](https://github.com/ManageIQ/manageiq/pull/12773).
+
+In the Service UI, RBAC is now enabled on [Orchestration Templates](https://github.com/ManageIQ/manageiq-ui-service/pull/576)
+
+The classic UI toolbar refactoring is moving along with lots of PRs merged, the full list can be found [here](https://github.com/ManageIQ/manageiq/issues/6259).
+
+
+## Fixed
+
+[JoeR](https://github.com/jrafanie) fixed a failure that would occur after starting new puma workers and try to gracefully restart apache.
+In his [PR](https://github.com/ManageIQ/manageiq/pull/14007), Apache is configured to boot the ports puma workers are bound to (from STARTING_PORT to the maximum worker count 
+port 3000 to 3009 if max worker count is 10) as balancer members.
+
+[Libor](https://github.com/lpichler) created a fix in this [PR](https://github.com/ManageIQ/manageiq/pull/14229) to allow regex for a
+MiqExpression::Field containing numbers in associations.
+
+## New
+
+[Rails 5.0.2.](Support Rails 5.0.2) is now supported.
+
+Check out [Ladas]() Target and Target Collections abstractions [here](#https://github.com/ManageIQ/manageiq/pull/14249).
```

Comment:
Should that be `[Ladas](https://github.com/Ladas)` and `[here](https://github.com/ManageIQ/manageiq/pull/14249)` (there's an extra `#`)